script.js

```diff
@@ -0,0 +1,96 @@
+// menu bar
+const toggle = document.getElementsByClassName('check-btn')[0];
+const navLi = document.getElementsByClassName('nav-options')[0];
+const commentSec = document.getElementById('featured');
+
+toggle.addEventListener('click', () => {
+  navLi.classList.toggle('active');
+});
+
+function myFunction(x) { // eslint-disable-line no-unused-vars
+  x.classList.toggle('fa-xmark');
+}
+
+// featured section dynamic html
+
+const data = [
+  {
+    name: 'Alex Sintek',
+    title: 'Pop singer',
+    img: 'Alex.jpg',
+    comment: 'So glad I could participate on this event, I will gladly go to the next one and sing some songs',
+  },
+  {
+    name: 'Andrea Belocia',
+    title: 'Comedian',
+    img: 'Andrea.jpg',
+    comment: 'Best event ever, Im so glad I could participate and give my little contribution for those poor animals.',
+  },
+  {
+    name: 'Rosa Melano',
+    title: 'Dog Lover',
+    img: 'Rosa.jpg',
+    comment: 'Im so happy for this kind of events, I hope next year more peaple can come and participate so they can recover more founds and help those little angels',
+  },
+  {
+    name: 'Simon Rodriguez',
+    title: 'Veterinarian',
+    img: 'SimonR.jpg',
+    comment: 'I am just so happy that there were that many people in this event, hope those beautiful animals can soon find a loving roof with some loving owners',
+  },
+  {
+    name: 'Stefan Suarex',
+    title: 'College Student',
+    img: 'Stefan.jpg',
+    comment: 'Got a new dog thanks to this event, ever since I adopted him my life and his changed completly, I cant belive such a good soul needed to be rescued.',
+  },
+  {
+    name: 'Gabriela Jadán',
+    title: 'Doctor',
+    img: 'gabriela.jpg',
+    comment: 'What a great event, food was delicius, concert was so fun, and I came back home with a beautiful dog and a lovely cat. I will give my best forr giving them the life they deseerve.',
+  },
+];
```

Comment:
Fantastic 🔥 The list of speakers used to generate the HTML is assigned to a variable as an array or object. The same list also contains an speaker object for each that holds their specific data.